cmd/manager/main.go

```diff
@@ -137,6 +142,42 @@ func main() {
 		os.Exit(1)
 	}
 
+	//create client that will be used for retrieving the host operator secret
+	cl, err := client.New(cfg, client.Options{})
+	if err != nil {
+		log.Error(err, "Unable to create new client")
+		os.Exit(1)
+	}
+
+	//Get webhook server deployment
+	opNS, err := k8sutil.GetOperatorNamespace()
+	namespacedName := types.NamespacedName{
+		Namespace: opNS,
+		Name:      server.WebhookServerDeploymentName,
+	}
+	webhookDeployment := &appsv1.Deployment{}
+	err = cl.Get(context.TODO(), namespacedName, webhookDeployment)
+	if err != nil {
+		log.Error(err, "Unable to get webhook deployment")
+		os.Exit(1)
+	}
+
+	//Add restart timestamp
+	if webhookDeployment.Spec.Template.Annotations == nil {
+		webhookDeployment.Spec.Template.Annotations = map[string]string{}
+	}
+	now := time.Now()
+	webhookDeployment.Spec.Template.Annotations[constants.RestartAtAnnotation] = strconv.FormatInt(now.UnixNano(), 10)
```

Comment:
I think it can be done in simpler way, if we just do it in which already update webhook server on each https://github.com/devfile/devworkspace-operator/blob/main/pkg/webhook/deployment.go

Since we have rolling update for webhook server, I think we should not care that webhook server will be restarted on each controller restart.

Reply:
Ohhhhh! okay, that makes it easier, there is already the client and the deployment in deployment.go. I've updated the PR to add the annotation in deployment.go. Thanks @sleshchenko